task/validate-fbc/0.1/README.md

```diff
@@ -16,4 +16,5 @@ Ensures file-based catalog (FBC) components are uniquely linted for proper const
 |TEST_OUTPUT|Tekton task test output.|
 |RELATED_IMAGES_DIGEST|Digest for attached json file containing related images|
 |IMAGES_PROCESSED|Images processed in the task.|
+|OCP_VERSION|OCP version of FBC image.|
```

Comment:
Should `RENDERED_CATALOG_DIGEST ` be added here too?

Also, could you have the changes to validate-fbc in a separate commit?

An afterthought. We should be using konflux-test helper functions to get OCP_VERSION and rendered catalog. Tekton results have a size limit and this approach wouldn't be scalable.